### PR TITLE
Icons for ema

### PR DESCRIPTION
**Pull request by dmcalester**
This PR should remain open for the time being as icons are worked on for EMA.

See further comments in Jira ticket https://rocketcom.atlassian.net/secure/RapidBoard.jspa?rapidView=32&projectKey=ASTRO&modal=detail&selectedIssue=ASTRO-540&search=icon

### CODE REVIEW

**Comment by sknep** [comment]
Is this a draft PR? You can use the draft state in github if it's not ready to merge (based on the comment about it staying open)

It's appropriate to add all new icons to the array in the icon story: https://github.com/RocketCommunicationsInc/astro-components/blob/next/stories/rux-icons-and-symbols.stories.js#L33

**Comment by sknep** [comment]
Sorry, didn't see that you'd already added them to the story. As this grows, should we sort the array by name? Also, we should add notes about the new icons in the [changelog at the end of the Readme](https://github.com/RocketCommunicationsInc/astro-components/blob/next/src/components/rux-icon/README.md#revision-history) before merge.

File: stories/rux-icons-and-symbols.stories.js [hunk 1]
```diff
@@ -58,10 +58,32 @@ export const AllIcons = () => {
...
+    'star',
+    'up',
+    'down',
+    'back',
+    'forward',
+    'hamburger',
+    'delete',
+    'help',
+    'visibility',
+    'visibility-off',
```
Comment (sknep) [inline stories/rux-icons-and-symbols.stories.js:81]: I think these are backwards?

**Comment by sknep** [comment]
I'm still of the opinion that we should add notes about the new icons in the changelog at the end of the Readme before merge here: https://github.com/RocketCommunicationsInc/astro-components/blob/next/src/components/rux-icon/README.md#revision-history

Even if it's just:

##### **4.1**
- Added new icons

File: static/icons/astro.svg [hunk 2]
```diff
@@ -85,4 +85,49 @@
...
   <path id="solar" fill-rule="evenodd" d="M118.153846,14 C123.591713,14 128,18.4225104 128,23.877953 L128,103.122047 C128,108.57749 123.591713,113 118.153846,113 L9.84615385,113 C4.40828718,113 0,108.57749 0,103.122047 L0,23.877953 C0,18.4225104 4.40828718,14 9.84615385,14 L118.153846,14 Z M32,29 L15,29 L15,59 L32,59 L32,29 Z M59,29 L42,29 L42,59 L59,59 L59,29 Z M86,29 L69,29 L69,59 L86,59 L86,29 Z M113,29 L96,29 L96,59 L113,59 L113,29 Z M32,68 L15,68 L15,98 L32,98 L32,68 Z M59,68 L42,68 L42,98 L59,98 L59,68 Z M86,68 L69,68 L69,98 L86,98 L86,68 Z M113,68 L96,68 L96,98 L113,98 L113,68 Z"/>
+
+  <path id="notifications-filled" d="M53,128 C60.2875,128 66.25,122.092308 66.25,114.871795 L39.75,114.871795 C39.75,122.092308 45.64625,128 53,128 Z M92.75,88.6153846 L92.75,55.7948718 C92.75,35.6430769 81.885,18.7733333 62.9375,14.3097436 L62.9375,9.84615385 C62.9375,4.39794872 58.49875,0 53,0 C47.50125,0 43.0625,4.39794872 43.0625,9.84615385 L43.0625,14.3097436 C24.04875,18.7733333 13.25,35.5774359 13.25,55.7948718 L13.25,88.6153846 L0,101.74359 L0,108.307692 L106,108.307692 L106,101.74359 L92.75,88.6153846 Z" transform="translate(11)"/>
```
Comment (cskerbo) [inline static/icons/astro.svg:89]: Test comment.

File: static/icons/astro.svg [hunk 3]
```diff
@@ -85,4 +85,49 @@
...
+
+  <path id="play" d="M0 0L0 75 59 37.5z" transform="translate(43 26)"/>
+  <path id="pause"  d="M0,75 L21.3333333,75 L21.3333333,0 L0,0 L0,75 Z M42.6666667,0 L42.6666667,75 L64,75 L64,0 L42.6666667,0 Z" transform="translate(32 26)"/>
+  <path id="stop" d="M0 0L64 0 64 64 0 64z" transform="translate(32 32)"/>
+
+  <g id="refresh" fill-rule="evenodd">
+    <path fill-rule="nonzero" d="M94.5078174,33.484375 C86.7998749,25.78125 76.2213884,21 64.4734209,21 C40.9774859,21 22,40.01875 22,63.5 C22,86.98125 40.9774859,106 64.4734209,106 C84.3014384,106 100.833646,92.453125 105.564728,74.125 L94.5078174,74.125 C90.148843,86.503125 78.3477173,95.375 64.4734209,95.375 C46.8780488,95.375 32.5784866,81.084375 32.5784866,63.5 C32.5784866,45.915625 46.8780488,31.625 64.4734209,31.625 C73.2976861,31.625 81.1651032,35.290625 86.9061914,41.08125 L69.7892433,58.1875 L107,58.1875 L107,21 L94.5078174,33.484375 Z"/>
+  </g>
+
```
Comment (cskerbo) [inline static/icons/astro.svg:98]: Another test comment. Hi Sarah!

**Review by cskerbo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**